webapp/src/components/organisms/MainScreen/MainScreen.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import './MainScreen.css';
+import NavBar from '../../organisms/NavBar/index';
+
+/**
+ * This is the main App component. If this starts getting too big, remember to refactor and nest things!
+ * @returns
+ */
+
+/* TODO* is for future Signe to delete these "Sound Effects" after PR is approved since it was just for fun! */
+
+function MainScreen() {
+  return (
+    <div className="MainScreen">
+      <NavBar />
```

Comment:
I feel like we could design this app differently where instead of MainScreen we use a template or something, but again over-engineered for this scale of project